apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -211,10 +214,39 @@ export const getDayLteFilter: (dataIndex: DataIndexType) => FilterType = (dataIn
     }
 }
 
+const datesFromTextQuery = (text) => {
+    const possibleDates = text.match(/[0-9.]+/g) || []
```

Comment:
May this regexp be stored and reused at some common cases?

Reply:
This is too common regex but we will need to move it to constants - when we will face another ticket's dates display. Maybe based on user locale as different countries have different formats for dates